usb_protocol/types/descriptors/microsoft.py

```diff
@@ -103,15 +103,15 @@ class MicrosoftRequests(IntEnum):
 
 
 FeatureRegProperty = DescriptorFormat(
-    "wLength"             / construct.Rebuild(construct.Int16ul, 10 + this.wPropertyNameLength + this.wPropertyDataLength),
+    "wLength"             / construct.Rebuild(construct.Int16ul, 10 + 2 + 2*len_(this.PropertyName) + 2 + 2*len_(this.PropertyData)),
```

Comment:
Would personally add spaces around the * and potentially a bracket to disambiguate, but this seems fine otherwise.